python/plugins/processing/algs/gdal/contour.py

```diff
@@ -192,5 +192,62 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
 
         arguments.append(inLayer.source())
         arguments.append(output)
+        return arguments
+
+    def getConsoleCommands(self, parameters, context, feedback, executing=True):
+        arguments = self._buildArgsList(parameters, context, feedback, executing)
+        return [self.commandName(), GdalUtils.escapeAndJoin(arguments)]
+
+
+class contour_polygon(contour):
+
+    FIELD_NAME_MIN = 'FIELD_NAME_MIN'
+    FIELD_NAME_MAX = 'FIELD_NAME_MAX'
+    OUTPUT = 'OUTPUT'
+
+    def __init__(self):
+        super().__init__()
+
+    def initAlgorithm(self, config=None):
+        super().initAlgorithm(config)
+        # FIELD_NAME isn't used in polygon mode
+        self.removeParameter(contour.FIELD_NAME)
+
+        self.addParameter(QgsProcessingParameterString(self.FIELD_NAME_MIN,
+                                                       self.tr('Attribute name to use for the minimum elevation of contour polygons (if not set, no elevation attribute is attached)'),
```

Comment:
```suggestion
                                                       self.tr('Attribute name for minimum elevation of contour polygon'),
```

It's a bit more concise, and the existing description is longer than we generally use for parameters

Reply:
(same with the max attribute)